runtime/tests/interpreter/builtinfunctions_test.go

```diff
@@ -196,24 +196,28 @@ func TestInterpretToBigEndianBytes(t *testing.T) {
 			"-9223372036854775808": {128, 0, 0, 0, 0, 0, 0, 0},
 		},
 		"Int128": {
-			"0":                  {0},
-			"42":                 {42},
-			"127":                {127},
-			"128":                {0, 128},
-			"200":                {0, 200},
-			"-1":                 {255},
-			"-200":               {255, 56},
-			"-10000000000000000": {220, 121, 13, 144, 63, 0, 0},
+			"0":                  {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
+			"42":                 {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 42},
+			"127":                {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 127},
+			"128":                {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 128},
+			"200":                {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 200},
+			"-1":                 {255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255},
+			"-200":               {255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 56},
+			"-10000000000000000": {255, 255, 255, 255, 255, 255, 255, 255, 255, 220, 121, 13, 144, 63, 0, 0},
```

Comment:
Good idea to add more test cases! 👍 

Could you please also add similar new cases for the smaller signed integer types, i.e. `Int8|16|32|64`?